Migration.md

```diff
@@ -14,6 +14,24 @@ release will remove the deprecated code.
 1. Header files under `ignition/...` are deprecated and will be removed in future versions.
    Use `gz/...` instead.
 
+1. The following `IGN_` / `IGNITION_` prefixed macros are deprecated and will be removed in future versions.
+   Additionally, they will only be available when including the corresponding `ignition/...` header.
+   Use the `GZ_` prefix instead.
+   1. `IGN_PHYSICS_DECLARE_JOINT_TYPE`, `IGN_PHYSICS_DECLARE_SHAPE_TYPE`
+
+   1. `IGN_PHYSICS_OPERATEONSPECIFIEDDATA_TEMPLATES`, `IGN_PHYSICS_OPERATEONSPECIFIEDDATA_PREFIX`
+   1. `IGN_PHYSICS_MAKE_ALL_TYPE_COMBOS`
+   1. `IGN_PHYSICS_ADD_PLUGIN`
+   1. `IGN_PLUGIN_REGISTER_MORE_TRANS_UNITS` -> `GZ_**PHYSICS**_REGISTER_MORE_TRANS_UNITS`
+   1. `IGN_PHYSICS_REQUEST_FEATURES_MACRO`
+   1. `IGN_PHYSICS_CREATE_SELECTOR`
+   1. `IGNITION_PHYSICS_CONST_GET_ERROR` (hard-tocked, inside detail header)
+   1. `IGNITION_UNITTEST_EXPECTDATA_ACCESS` (hard-tocked, inside test and detail headers)
+   1. `IGNITION_PHYSICS_DEFINE_COORDINATE_SPACE` (hard-tocked, inside detail header)
+   1. `IGN_PHYSICS_MOCK_MACRO_HELPER`
+   1. `IGN_PHYSICS_MOCK_MACRO`
```

Comment:
These `MOCK` macros aren't being tick-tocked, they're not exposed to users, so no need to add to the migration guide

Reply:
https://github.com/gazebosim/gz-physics/pull/353/commits/c18e0f04d406ad066d91dbca97f2f4af63e16def